config/locales/en/providers.yml

```diff
@@ -338,6 +338,7 @@ en:
           size_hint: The maximum file size is 7MB. You can attach more than one file.
           warning: Warning
           warning_text: You must provide a complete statement now. You will not be able to upload a statement later.
+          file_not_uploaded_error: Upload the chosen file
```

Comment:
Might want to check this with Jim, but we have used 'selected' instead of 'chosen' elsewhere in the app. e.g. 

              file_empty: The selected file is empty
              file_too_big: The selected file must be smaller than %{size}MB
              file_virus: The selected file contains a virus

Reply:
I did check with Jim earlier and he said to go with 'chosen' but I can point this out to him - selected does seem more similar to other errors 🤔

Reply:
Jim agrees - have updated!